sonar-dotnet-shared-library/src/test/resources/opencover/coverage_branches.xml

```diff
@@ -1,4 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+This file was generated from the following code:
+
+namespace BranchCoveragePoc
+{
+    public class Calculator
+    {
+        public int Compute(int x, int y, int z)
+        {
+            return x < 2
+                ? y < 3
+                    ? z < 1
+                        ? 1
+                        : 2
+                    : 3
+                : 4;
+        }
+    }
+}
+
+Unit tests:
+
+using NUnit.Framework;
+
+namespace BranchCoveragePoc.Tests
+{
+    public class Tests
+    {
+        [Test]
+        public void Test1()
+        {
+            var sut = new Calculator();
+
+            Assert.That(sut.Compute(0, 0, 0).Equals(1));
+        }
+    }
+}
+
+OpenCover version 4.7.922.0
+
+Commands used:
+msbuild BranchCoveragePoc.sln
+OpenCover.Console.exe -target:"c:\Program Files\dotnet\dotnet.exe" -targetargs:"test" -output:coverage.xml -register:user -filter:"+[BranchCoveragePoc*]* -[BranchCoveragePoc.Tests]*" -oldStyle
```

Comment:
regarding `-oldStyle` - is this necessary?
if so, after doing the release we must update https://community.sonarsource.com/t/coverage-test-data-generate-reports-for-c-vb-net/9871

Reply:
and this should be taken into consideration when doing the tests with coverlet

Reply:
No, this is not needed and it's should not affect us. I removed it since it can be misleading.